style.css

```diff
@@ -2,650 +2,391 @@
+  gap: 8px;
+  position: absolute;
+  width: 255px;
+  height: 40px;
+  right: 190px;
+  top: calc(50% - 40px / 2);
+  z-index: 99;
+}
+
+#fram4 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  gap: 10px;
+  width: 87px;
+  height: 40px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+#portf {
+  width: 63px;
+  height: 20px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+#rectang {
+  position: absolute;
+  width: 100%;
+  height: 72px;
+  background: #3e3c3c;
+}
+
+#a {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  gap: 10px;
+  position: absolute;
+  width: 116px;
+  height: 40px;
+  left: 30px;
+  top: 16px;
+  z-index: 99;
+}
+
+#johndoe {
+  width: 92px;
+  height: 20px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+#about {
+  width: 45px;
+  height: 20px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+#contact {
+  width: 59px;
+  height: 20px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+#normal_btn {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  right: 142px;
+  top: calc(50% - 40px / 2);
+  border-radius: 8px;
+}
+
+#icons {
+  width: 24px;
+  height: 24px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+#union1 {
+  position: absolute;
+  width: 22px;
+  height: 18px;
+  left: 1px;
+  top: 3px;
+  background: #3e3c3c;
+}
+
+#vectorx {
+  position: absolute;
+  left: 25%;
+  right: 25%;
+  top: 33.33%;
+  bottom: 50%;
+  border: 2px solid #67798e;
+}
+
+#rec {
+  box-sizing: border-box;
+  position: absolute;
+  left: 4.17%;
+  right: 4.17%;
+  top: 12.5%;
+  bottom: 12.5%;
+  border: 2px solid #67798e;
+  border-radius: 4px;
+}
+
+#ul {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  width: 20px;
+  height: 98.96px;
+  left: -100px;
+  top: 535px;
+  z-index: 2;
+}
+
+li {
+  flex: 1;
+}
+
+#section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 120px 24px;
+  gap: 24px;
+  position: absolute;
+  width: 566px;
+  height: 739px;
+  left: 10%;
+  top: 72px;
+}
```

Comment:
- [ ] **Thank you for implementing the required changes by the previous code reviewer :1st_place_medal: :1st_place_medal:**
As suggested by the previous code reviewer, I think it would be better to add the proper background image  as given by the Figma design :joy: 
Remember it is important that you try to replicate the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) as closely as possible to give confidence to recruiters in your ability to produce exact business requirements :pray:

**Your design** | **Figma design**
| --- | --- |
![image](https://user-images.githubusercontent.com/102757126/205084037-aee4b645-8e94-4934-825f-c419a817bde0.png) | ![image](https://user-images.githubusercontent.com/102757126/205084279-2fe008f9-8dab-4a7f-8a60-81a2cf8f5659.png)